Submitted By:            Igor Živković
Date:                    2014-12-11
Initial Package Version: 0.5.8
Upstream Status:         Not submitted
Origin:                  Self
Description:             Removes dependency on coreutils (neither toybox nor
                         sbase nl support the -v option).

diff -Naur dash-0.5.9.orig/src/mkbuiltins dash-0.5.9/src/mkbuiltins
--- dash-0.5.9.orig/src/mkbuiltins	2015-06-11 08:31:48.000000000 +0200
+++ dash-0.5.9/src/mkbuiltins	2016-06-09 13:47:38.674974232 +0200
@@ -97,7 +97,7 @@
  */
 
 !
-sed 's/	-[a-z]*//' $temp2 | nl -b a -v 0 |
+sed 's/	-[a-z]*//' $temp2 | awk '{print NR-1,$0}' |
 	LC_ALL= LC_COLLATE=C sort -u -k 3,3 |
 	tr abcdefghijklmnopqrstuvwxyz ABCDEFGHIJKLMNOPQRSTUVWXYZ |
 	awk '{	printf "#define %s (builtincmd + %d)\n", $3, $1}'
